test/functional/fixtures/run-options/allow-multiple-windows/pages/cookie-synchronization/same-domain/iframe.html

```diff
@@ -0,0 +1,11 @@
+<div id="cookie"></div>
```

Comment:
Wrap this html to the `html` and `body` tag to it looks like a regular web page markup.